packages/ui/src/working-groups/queries/__generated__/workingGroups.generated.tsx

```diff
@@ -722,7 +722,7 @@ export type CountWorkingGroupWorkersQuery = {
 
 export type GetWorkingGroupOpeningsQueryVariables = Types.Exact<{
   where?: Types.InputMaybe<Types.WorkingGroupOpeningWhereInput>
-  order?: Types.InputMaybe<Array<Types.WorkingGroupOpeningOrderByInput> | Types.WorkingGroupOpeningOrderByInput>
+  order?: Types.InputMaybe<Types.WorkingGroupOpeningOrderByInput>
```

Comment:
This file is generated, it shouldn't be edited manually, or else the changes will be overridden every time someone runs `yarn workspace @joystream/pioneer queries:generate`